test/test_assert_that.py

```diff
@@ -141,3 +142,10 @@ def test_with_kwargs(self, assert_that, automation):
             assert_that(LIGHT).was_not.turned_off()
             automation.turn_off_light_with_transition(via_helper=True)
             assert_that(LIGHT).was.turned_off(transition=TRANSITION_DURATION)
+
+class TestSelectOption:
+    class TestViaService:
+        def test_option_is_set(self, assert_that, automation):
+            assert_that(INPUT_SELECT).was_not.set_to_option('new_option')
+            automation.select_option('new_option')
+            assert_that(INPUT_SELECT).was.set_to_option('new_option')
```

Comment:
Just one thing I noticed after answering your question above. Could you please add a test for the case when the option is selected via a service call instead of via the helper? :)